.github/workflows/e2e.yml

```diff
@@ -14,12 +14,12 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: "3.7"
-          cache: "pip"
 
       - name: Install deps
         run: |
-          pip install -r requirements.txt -r requirements_dev.txt && git submodule update --init && pip install -r kiliautoml/utils/ultralytics/yolov5/requirements.txt
-
+          pip install -r requirements_ci.txt -r requirements_dev.txt && git submodule update --init && pip install -r kiliautoml/utils/ultralytics/yolov5/requirements.txt
+      - name: Install detectron2
+        run: pip install git+https://github.com/facebookresearch/detectron2.git
```

Comment:
why not adding `git+https://github.com/facebookresearch/detectron2.git` that into the requirements_ci.txt file?

Reply:
see also the comment in the `requirements_ci.txt`

Reply:
I do not understand either, but it was not working in the ci

Reply:
I needed to separate requirements_ci.txt containing almost every package and detectron2 for an unknown reason